android/package.json

```diff
@@ -26,7 +26,7 @@
 		"android platform tools": "27.x",
 		"android tools": "<=26.x",
 		"android ndk": ">=r11c <=r16c",
-		"node": ">=4.0 <=7.x",
+		"node": ">=6.0 <=8.x",
```

Comment:
So we don't need `<= 10.x` here?

Reply:
Left out on purpose as this PR doesn't introduce full Node 10 support. This + #10089 + some further testing and I'd feel comfortable claiming support

Reply:
Alright. I think it should be added and tested with this ticket, as it's all about Node 10 anyway.

Reply:
Want me to cherry-pick your commit across to here?

Reply:
Isn't it a different change? My one only updated ioslib and it's compatibility, but I think the CLI will warn if the package.json in the iphone/android dir of the SDK are not updated as well.

Reply:
@ewanharris  I've merged Hans' ioslib update PR after fixing ioslib/node-ios-device, so this can now be tested for Node 10 support.